mcs/class/System/Mono.Btls/X509CertificateImplBtls.cs

```diff
@@ -334,10 +331,7 @@ void MustFallback ()
 				ThrowIfContextInvalid ();
 				if (subjectName == null) {
 					using (var xname = x509.GetSubjectName ()) {
```

Comment:
Same, I'm using the managed CoreFX PAL for this, so we don't have to worry about this.